desktop_server/src/com/mygdx/game/server/di/Names.java

```diff
@@ -0,0 +1,8 @@
+package com.mygdx.game.server.di;
+
+public final class Names {
+  public static final String MAP_GENERATOR_LIST = "Map Generator List";
```

Comment:
To gdzieś wogóle używasz? No i też nazwa słaba

Reply:
a może nie używam, ale nazwa nie jest słaba, bo to było używane do robienia @Named dependencies i to miało taki sens, że nie trzeba było wpisywać ręcznie stringa wszędzie gdzie coś było injectowane, tylko IDE samo podpowiadało, ale usunę jak teraz tego nie używamy